bot/pagination.py

```diff
@@ -345,27 +333,22 @@ async def paginate(
                 else:
                     embed.set_footer(text=f"Page {current_page + 1}/{len(paginator.pages)}")
 
-                await message.edit(embed=embed)
-
-            if reaction.emoji == RIGHT_EMOJI:
-                await message.remove_reaction(reaction.emoji, user)
-
-                if current_page >= len(paginator.pages) - 1:
-                    log.debug("Got next page reaction, but we're on the last page - ignoring")
-                    continue
-
-                current_page += 1
-                log.debug(f"Got next page reaction - changing to page {current_page + 1}/{len(paginator.pages)}")
-
-                embed.description = paginator.pages[current_page]
-
-                if footer_text:
-                    embed.set_footer(text=f"{footer_text} (Page {current_page + 1}/{len(paginator.pages)})")
-                else:
-                    embed.set_footer(text=f"Page {current_page + 1}/{len(paginator.pages)}")
-
-                await message.edit(embed=embed)
+                try:
+                    await message.edit(embed=embed)
+                except discord.HTTPException as e:
+                    if e.code == 50083:
+                        # Trying to act on an archived thread, just ignore and abort
+                        break
+                    else:
+                        raise e
```

Comment:
```suggestion
                except discord.HTTPException:
                    if e.code == 50083:
                        # Trying to act on an archived thread, just ignore and abort
                        break
                    else:
                        raise
```

Reply:
Did you mean to remove the `as e` here? We need it to get the code

Reply:
oh I'm dumb, right